.github/workflows/pull-request.yml

```
@@ -295,7 +295,81 @@ jobs:
+          # Cleanup
+          if ! docker-compose -f docker-compose.dev.yaml down -v; then
+          echo "Failed to cleanup containers"
+          exit 1
+          fi
+    env:
+      HEALTH_CHECK_URL: http://localhost:4000
+      COMPOSE_PROJECT_NAME: pr-${{ github.event.pull_request.number }}
+      MONGO_DB_URL: mongodb://mongodb:27017/talawa-test-db
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
+      ACCESS_TOKEN_SECRET: ${{ env.ACCESS_TOKEN_SECRET }}
```

actionlint (1.7.3)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)